app/concepts/base/use_case/base.rb

```diff
@@ -0,0 +1,14 @@
+module UseCase
+    module Base
+
+        class Base
```

Comment:
I don't see the reason to create the base use_case class. You just initialize instance with repository here. Probably particular use_cases will be initialized with different repositories. Also, you don't need to initialize the use_case class always with repositories.

Reply:
@baniak Well, I wanted to reduce copy/paste of constructors, so I created base. Well, I could make one repository usecase for all, but what I wanted, was readibility for myself. Like This way, I can orginize my work, because I know where everything is.

Reply:
Your main goal should be writing code that is easily understood by other developers. Software development is mainly teamwork.

Reply:
@baniak66
Ok, that was the case when I was coding it. I asked myself a question what is the best way for me and others to read it.